src/platform/icelake/include/platform/lib/shim.h

```diff
@@ -30,30 +30,30 @@
 
 
 /* DIPCTDR */
-#define IPC_DIPCTDR_BUSY	(1 << 31)
+#define IPC_DIPCTDR_BUSY	BIT(31)
 #define IPC_DIPCTDR_MSG_MASK	0x7FFFFFFF
 
-/* DIPCTDA */
-#define IPC_DIPCTDA_BUSY	(1 << 31)
+/* DIPCTDA - ack from target to initiator */
+#define IPC_DIPCTDA_DONE	BIT(31)
 #define IPC_DIPCTDA_MSG_MASK	0x7FFFFFFF
 
-/* DIPCTE */
-#define IPC_DIPCTE_MSG_MASK	0x3FFFFFFF
+/* DIPCTDD */
+#define IPC_DIPCTE_MSG_MASK	0xFFFFFFFF
```

Comment:
You can use `MASK(x, y)` macro.

Reply:
Aren't we going to define those masks in a shared way with the kernel for each generation, so that the intel-specific parts of the SOF driver can use those masks to generate the relevant messages?

Also I see no specific reason why this is Icelake specific, should be common for all cAVS 1.8 platforms, no?

Reply:
Replaced with `MASK()` and made the defs common for cAVS 1.8+ shims (offsets remained platform specific).
The new sideband-ipc file contains the defs which could be shared with the kernel (names of the regs are a bit different on the kernel side though).